pwiz_tools/Skyline/Model/Lib/LibResources.resx

```diff
@@ -344,4 +344,7 @@
   <data name="NistLibraryBase_GetMod_Unknown_modification__0__at_line__1_" xml:space="preserve">
       <value>Unknown modification {0} at line {1}</value>
   </data>
+  <data name="NistLibraryBase_CreateCache_Missing_details_for__0__at_line__1___ignored" xml:space="preserve">
+      <value>Missing details for {0} at line {1}, ignored</value>
```

Comment:
Could this warning message be rephrased so that it is more clear how big of a thing is being ignored?

Is just the line which has the missing details being ignored, or is the entire spectrum for the molecule being excluded from the spectral library?

Reply:
OK, updated to say that the entry is being ignored